Frontend/CO.CDP.OrganisationApp/ExceptionMiddleware.cs

```diff
@@ -14,9 +16,35 @@ public async Task Invoke(HttpContext context)
         {
             await this._next.Invoke(context);
         }
-        catch
+        catch (Exception ex)
         {
-            context.Response.Redirect("/error");
+            await HandleExceptionAsync(context, ex);
+        }
+    }
+    private static Task HandleExceptionAsync(HttpContext context, Exception exception)
+    {
+        context.Response.ContentType = "application/json";
+
+        switch (exception)
```

Comment:
I think 4xx errors shouldn't be handled this way, 
e.g. Unauthorised response from API should directly go to error page.
Not found response should be handled from the page itself rather propagated to the general exception middleware.
And certain Bad request (e.g. duplicate org name) should also be handles from the page and allow user to retry.

Reply:
ExceptionMiddleware is now removed and we are handling the error on the page level based on the API.